### DISCUSSION TITLE
Do not print the logo

### DISCUSSION THREAD

**Discussion by AndreyMZ** [projectdiscovery/nuclei#4659] [Q&A]
### Please describe your feature request:

Add a command line option to disable printing the following logo (and nothing else).
OR
Just remove the logo unconditionally.

```
                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v3.1.5

                projectdiscovery.io 
```

### Describe the use case of this feature:

The logo does not have any value, it only wastes space on a screen. So, it could be always useful to get rid of it. Especially it is useful when one needs to share both the command and it's the output, e.g.:

```
$ docker run -v .:/wrk -w /wrk -it projectdiscovery/nuclei:latest -disable-update-check -templates example.template.yaml -target host.example:123 


                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v3.1.5

                projectdiscovery.io

[ERR] Could not read nuclei-ignore file: open /root/.config/nuclei/.nuclei-ignore: no such file or directory
[INF] Current nuclei version: v3.1.5 (outdated)
[INF] Current nuclei-templates version:  (latest)
[WRN] Scan results upload to cloud is disabled.
[INF] New templates added in latest release: 0
[INF] Templates loaded for current scan: 1
[WRN] Executing 1 unsigned templates. Use with caution.
[INF] Targets loaded for current scan: 1
[INF] No results found. Better luck next time!
```

**Comment by ehsandeep** [comment]
@AndreyMZ there is `-silent` option for it.

**Reply by AndreyMZ** [reply]
@ehsandeep,

Have you tried it?

```
$ docker run -v .:/wrk -w /wrk -it projectdiscovery/nuclei:latest -disable-update-check -templates example.template.yaml -target host.example:123 -silent
[FTL] Could not run nuclei: no templates provided for scan
```

Have you read [the documentation](https://docs.projectdiscovery.io/tools/nuclei/running#nuclei-flags) at least?

>      -silent                       display findings only

The `-silent` option hides not only the logo, but also the log records (at least INF, WRN, and ERR). I wrote it specifically:

> Add a command line option to disable printing the following logo (**and nothing else**)
